### PR TITLE
Allow "wrapping" datetime spec

### PR DESCRIPTION
**Pull request by glennji**
With the current implementation, we can't (easily) specify times that wrap over midnight e.g. set a downtime from 8pm through to 6am every weeknight; same for day ranges that want to "wrap" into the following week e.g. Friday through to Tuesday. This patch handles both.

NOTE1: I realize it could also be handled with comma-delimited date-specs, but that feels messy to me.

NOTE2: Also renamed cmd.py to command.py to avoid clash with Pdb.

### CODE REVIEW

File: tests/test_time.py [hunk 1]
```diff
@@ -17,6 +17,9 @@ def test_time_spec():
 
     dt = datetime(2017, 11, 26, 15, 33)
     assert matches_time_spec(dt, 'Sat-Sun 15:30-16:00 UTC')
+    assert matches_time_spec(dt, 'Sat-Sun 15:30-08:00 UTC')
```
Comment (hjacobs) [inline tests/test_time.py]: isn't this a bit ambiguous? is the time range including 08:00 UTC on Monday or not?

**Comment by hjacobs** [comment]
Can you explain this behavior in the README? I think it's not completely intuitive/straightforward and open for interpretation (Sat-Sun 22:00-08:00 --> does it include Monday until 08:00 even so I did not specify Monday?)

File: tests/test_time.py (hunk 1 above)
Comment (glennji) [reply tests/test_time.py]: Yeah, as written it does include the hours after midnight on Monday morning. I agree it's ambiguous. What might be a better behaviour do you think?

Comment (glennji) [reply tests/test_time.py]: e.g. 'Wed-Sun 15:00-08:00 UTC' definitely could mean "between 3pm and 8am every day until 8am Sun". I'd be happy with that. If you are, I'll update the PR and include explanation in the README.

Comment (glennji) [reply tests/test_time.py]: My particular use-case is along the lines of "downscale (dev, test) from 6pm 'til 7am every weeknight", and it confused me why it never happened! :-)

**Comment by hjacobs** [comment]
@glennji are you still interested in this PR?

**Comment by hjacobs** [comment]
There seems to be no reaction. Closing this PR.

**Pull request closed without merging**